task2/Dockerfile

```diff
@@ -0,0 +1,24 @@
+# Use base image with python
+FROM python:3.8
```

Comment:
If you want to complete "Junior" part - you need:
1. Change the line to `FROM: ubuntu:20.04` and make it working again
2. Change BRANCH=mysql and make it work with mysql database. Configure the mysql service as this: user=mysql, database=mysql, password=mysql